packages/k8s/k0s/build.yaml

```diff
@@ -0,0 +1,40 @@
+requires:
+- name: "toolchain-go-ubuntu"
+  category: "development"
+  version: ">=0"
+env:
+  - K0S_VERSION=v{{ regexReplaceAll "\\+\\d+$" .Values.version "" }}+k0s.{{.Values.k0s_version}}
+  {{$arch:=(default "amd64" .Values.arch)}}
+  {{ if eq $arch "arm" }}
+  - ARCH=arm64
+  {{ else }}
+  - ARCH={{ $arch }}
+  {{ end }}
+steps:
+  - curl -sfL https://get.k0s.sh > installer.sh
+  {{ if eq .Values.name "k0s-openrc" }}
+  - touch /sbin/openrc-run && chmod +x /sbin/openrc-run
+  {{ else }}
+  - touch /bin/systemctl && chmod +x /bin/systemctl
```

Comment:
I think we need the real systemd installed so it can access the systemctl command directly, otherwise it fails to install it:

```
       #15 0.508 Error: failed to install k0s service: failed to install service: "systemctl" failed: fork/exec /usr/bin/systemctl: exec format error
```